app/assets/javascripts/discourse/app/lib/text.js

```diff
@@ -68,6 +68,12 @@ export function sanitizeAsync(text, options) {
   });
 }
 
+export function parseAsync(md, options) {
```

Comment:
A philosophical question 😃

Should we (re-)expose functions like this one through the plugin api? Or do we consider `lib` public surface as well? 🤔

Reply:
Yes that's a good question, probably not one we want to answer in this PR but I would tend to agree this should be exposed in the plugin-api